README.md

```diff
@@ -30,7 +30,7 @@ Read the full [documentation][1] and check out [this example iOS app][14] and th
 
 ## Supported Platforms
 
-* Swift 5.3+
+* Swift 5.3+ (for Swift 5.1 support, use the [`swift-tools-5-1` branch](https://github.com/Peter-Schorn/SpotifyAPI/tree/swift-tools-5-1))
```

Comment:
Change this to:

(for Swift 5.1 and 5.2 support, use the [`swift-tools-5-1`](https://github.com/Peter-Schorn/SpotifyAPI/tree/swift-tools-5-1) branch)